lib/lbt/bundle/Builder.js

```diff
@@ -440,6 +484,8 @@ class BundleBuilder {
 				// because whitespace of HTML <pre> should be preserved (should only happen rarely)
 				if (!xmlHtmlPrePattern.test(fileContent)) {
 					fileContent = pd.xmlmin(fileContent.toString(), false);
+					fileContent = fileContent.replace(/\n/g, "");
```

Comment:
Please remove these two replacement steps. They can't differentiate between whitespace between tags and whitespace inside e.g. string literals in expression bindings.  It is hard to imagine how such unescaped whitespace could be used, but not modifying those parts of the view is a matter of principle.

Reply:
I changed this because it was actually difference with the current tool :) But I can remove it for now